jsapp/js/constants.es6

```diff
@@ -276,6 +276,11 @@ export const QUESTION_TYPES = Object.freeze({
     faIcon: 'fa-lato-range',
     id: 'range',
   },
+  xml_external: {
+    label: t('External XML'),
+    faIcon: 'fa-external-link',
+    id: 'xml-external',
```

Comment:
`id` should be `xml_external`, so if this happens we don't have a bug:

```js
// in case that row is type `xml_external`
if (QUESTION_TYPES.xml_external.id === row.type) {…
```

Reply:
Unfortunately it needs to be this way for XLSForm syntax :cry: